### PR TITLE
Fix wrong work field name in sql

### PR DESCRIPTION
**Pull request by weijunnla**
@m-r-c @yetti @shuangzhou

### CODE REVIEW

**Comment by nlabuilder** [comment]
Build finished.

File: src/main/java/amberdb/graph/dao/AmberDao.java [hunk 1]
```diff
@@ -404,7 +404,7 @@
 			+" commentsInternal TEXT,"
 			+" commercialStatus VARCHAR(63),"
 		    +" commercialStatusReasonForChange VARCHAR(63),"
-			+" commercialStatusReasonForChangeSum VARCHAR(63),"
+			+" commercialStatusReasonForChangeSum VARCHAR(126),"
```
Comment (shuangzhou) [inline src/main/java/amberdb/graph/dao/AmberDao.java]: Should this field have a larger size maybe 255?

Comment (weijunnla) [inline src/main/java/amberdb/graph/dao/AmberDao.java]: Yes

**Comment by weijunnla** [comment]
@yetti These fields store text as the template shown: Lookup value of Primary field (e.g. Access Access lookup value) + Lookup value of Reason for Change  + (change by username on dd/MM/yyyyy). I think VARCHAR should be ok.

**Pull request merged**